tasks-by-users/Tauqeer-Ahmed-99/04. JS Rock, Paper, Scissors/script.js

```diff
@@ -0,0 +1,71 @@
+const btn = document.getElementById("btn");
+const SELECTION_ROCK = "ROCK";
+const SELECTION_PAPER = "PAPER";
+const SELECTION_SCISSORS = "SCISSORS";
+const DEFAULT_SELECTION = "PAPER";
+
+let userIp;
+let compSelection;
+
+const userInput = () => {
+  userIp = prompt(
+    `Rock, Paper or Scissors? 
+  You can Also choose:
+  1 . for ${SELECTION_ROCK}
+  2 . for ${SELECTION_PAPER}
+  3 . for ${SELECTION_SCISSORS}`,
+    ""
+  ).toUpperCase();
+  if (userIp == 1) {
+    userIp = SELECTION_ROCK;
+  } else if (userIp == 2) {
+    userIp = SELECTION_PAPER;
+  } else if (userIp == 3) {
+    userIp = SELECTION_SCISSORS;
+  }
+
+  if (
+    userIp !== SELECTION_ROCK &&
+    userIp !== SELECTION_PAPER &&
+    userIp !== SELECTION_SCISSORS
+  ) {
+    userIp = DEFAULT_SELECTION;
+    alert(
+      `You typed an invalid input, taking default input as ${DEFAULT_SELECTION}.`
+    );
+  }
```

Comment:
Could u convert lines 19 to 36 to use switch case instead of if else, would look much cleaner

Reply:
Okay I will do it asap @AlishaPrasad ma'am.